src/main/scala/nl.knaw.dans.easy.validatebag/rules/metadata/package.scala

```diff
@@ -527,6 +533,25 @@ package object metadata extends DebugEnhancedLogging {
     }
   }
 
+  // borrowed from easy-split-multi-deposit/MultiDepositParser.scala
```

Comment:
Maybe a more readable example to reuse:
https://github.com/DANS-KNAW/easy-convert-bag-to-deposit/blob/7e3bd6bdd5a9ab01dc646c580bb459653c27fc42/src/main/scala/nl.knaw.dans.easy.bag2deposit/UserTransformer.scala#L27-L34

With `r.getRecordNumber` you won't need zip-with-index, what is info you are not using anyway.

Reply:
done